src/sentry/api/serializers/rest_framework/dashboard.py

```diff
@@ -296,9 +298,57 @@ class DashboardDetailsSerializer(CamelSnakeSerializer):
     id = serializers.CharField(required=False)
     title = serializers.CharField(required=False, max_length=255)
     widgets = DashboardWidgetSerializer(many=True, required=False)
+    projects = ListField(child=serializers.IntegerField(), required=False, default=[])
+    environment = ListField(child=serializers.CharField(), required=False, allow_null=True)
+    range = serializers.CharField(required=False, allow_null=True)
+    start = serializers.DateTimeField(required=False, allow_null=True)
+    end = serializers.DateTimeField(required=False, allow_null=True)
```

Comment:
nit: Should we mirror PageFilters type with these names? so rename range -> period? But discover uses range so I'm ok with either. 
q: Should we also store utc?
```
datetime: {
    end: DateString;
    period: string | null;
    start: DateString;
    utc: boolean | null;
  }
```